packages/lesswrong/lib/collections/posts/views.ts

```diff
@@ -1218,3 +1219,23 @@ Posts.addView("stickied", (terms: PostsViewTerms) => (
     },
   }
 ));
+
+
+Posts.addView("nominatablePostsByVote", (terms: PostsViewTerms) => {
+  return {
+    selector: {
+      _id: {$in: terms.postIds},
+      userId: {$ne: terms.userId},
+      isEvent: false
+    },
+    options: {
+      sort: {
+        baseScore: -1
+      }
+    }
+  }
+})
+ensureIndex(Posts,
+  augmentForDefaultView({ postIds:1 }),
```

Comment:
This isn't a field on `Posts`. The index should be covered by the `_id` index anyways.